docs/build/interoperate-qiskit-qasm3.mdx

```diff
@@ -81,6 +81,10 @@ circuit.draw("mpl")
 ```
 ![output](/images/build/interoperate-qiskit-qasm3/qasm3circ.png)
 
+## Export to OpenQASM 3
```

Comment:
@jakelishman I added this rather lame section as a placeholder. Maybe when you have more bandwidth we can spin up a better section or at least add an example (I'm assuming we don't want to just copy the example from the API docs?)

Reply:
Update: @javabster guided me through figuring out examples, and my efforts are now added in (needs reviewing).